docs/add-ons/managed-add-ons.md

```diff
@@ -37,6 +38,7 @@ EKS managed add-ons can be enabled via the following.
     resolve_conflicts        = "OVERWRITE"
     namespace                = "kube-system"
     service_account_role_arn = ""
+    preserve                 = true # This option makes the add-on a self-managed add-on, rather than an Amazon EKS add-on. There is no downtime for the add-on.
```

Comment:
since this is the managed addon example, we should prob show these as false in the docs. Also, can you expand on what you mean by no downtime?

Reply:
Also should we call this something like 'enable_self_manage' or something? Just to make it super clear what this actually does. We can add more info in the docs.

Reply:
@kenerwin88 Updated the PR with NOTE
@askulkarni2 This is actually an EKS Add-on built in option which we aren't leveraging so far. By default this option option is false. I have made it true for critical add-ons.